### DISCUSSION TITLE
403 forbidden

### DISCUSSION THREAD

**Discussion by SteveGanser** [weather-gov/api#581] [Q&A] [answered]
From what I have read, this is probably an MDL question, but I cannot find out how to contact them.  

My company utilizes a numb of your weather services.  Early today, these started getting 403's... example:  https://graphical.weather.gov/xml/SOAP_server/ndfdSOAPclientByDay.php?lat=30.13&lon=-96.22&format=12+hourly&startDate=2022-8-30&numDays=7&Submit=Submit&wait=32
 
It gets: <error>
<h2>ERROR</h2>
<pre>HTTP Error: Unsupported HTTP response status 403 Forbidden (soapclient->response has contents of the response)</pre>
</error>

Which has worked for 6 + months.  We are using a user-agent, in vb.net:
Dim myWebClient As New WebClient()
			myWebClient.Headers("Accept") = "application/x-ms-application, image/jpeg, application/xaml+xml, image/gif, image/pjpeg, application/x-ms-xbap, application/x-shockwave-flash, application/vnd.ms-excel, application/vnd.ms-powerpoint, application/msword, */*"
			myWebClient.Headers("User-Agent") = "Bushel/Farmcentric sganser@bushelpowered.com"
			ServicePointManager.Expect100Continue = True
			ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
			xmlForecast = myWebClient.DownloadString(_forecastUrl)

Not seeing any Reference number in the response.  Any info, or alternate discussion to check would be greatly appreciated.

**Comment by sullynole** [comment, marked as answer]
https://vlab.noaa.gov/web/mdl/ndfd-support

**Comment by dwhitemv25** [comment]
At the moment (8/30/2022 14:48PM PDT here), there's some messages from NCO reporting issues in a datacenter that is impacting some services. Updates are posted here: https://www.nco.ncep.noaa.gov/status/messages/
